packages/core/lib/commands/opcode/run.js

```diff
@@ -29,15 +30,15 @@ module.exports = async function (options) {
   }
   const opcodes = CodeUtils.parseCode(bytecode, numInstructions);
 
-  const indexLength = (opcodes.length + "").length;
+  const lastPCByteLength = Conversion.toBytes(
+    opcodes[opcodes.length - 1].pc
```

Comment:
Oh wait!  Sorry, one more thing!  I just realized this might cause problems if `opcodes` is an empty array.  Could you throw in something to handle that case?  Thank you!

Reply:
Oh right, I'll fix that